prog/kubernetes/provision_kubernetes_node.rb

```diff
@@ -228,6 +228,7 @@ def before_run
 
   label def approve_new_csr
     kubernetes_cluster.sshable.cmd("sudo kubectl --kubeconfig /etc/kubernetes/admin.conf get csr | awk '/Pending/ && /kubelet-serving/ && /'\"#{node.name}\"'/ {print $1}' | xargs -r sudo kubectl --kubeconfig /etc/kubernetes/admin.conf certificate approve")
+    kubernetes_cluster.incr_sync_worker_dns_config
```

Comment:
this could be unrelated but shall we do incr_sync_worker_mesh here as well as on KubernetedNodeNexus#destroy?